tests/Unit/Core/ConstraintValidator/CleanHtmlValidatorTest.php

```diff
@@ -56,8 +56,20 @@ public function testItFailsWhenJavascriptEventsAreGiven()
         ;
     }
 
+    public function testItFailsWhenIframeIsGiven()
+    {
+        $htmlTag = '<iframe src="catvideo.html" /></iframe>';
```

Comment:
You should add more tests, regarding the condition, you have to test with form, object, embed, etc. With or without spaces  ̀<    iframe   >`, What about `<script>`?

Reply:
Added more tests.

<script> tag is already tested by original test and it fails regardless if iframe can be used or not. Or you meant something else?